config/routes.rb

```diff
@@ -1,7 +1,8 @@
 Rails.application.routes.draw do
   devise_for :users
   resources :users, only: %i[index show] do
-    resources :posts, only: %i[index new create show]
+    resources :posts, only: %i[index new create show] do # rubocop:todo Lint/EmptyBlock
+    end
```

Comment:
- [ ] Well implementation. However I cannot see your api endpoints. Kindly add them as you described in controllers.


> Example:
> 
> http://localhost:3000/api/users/...
>  or ..api/users/1/posts..
>

Reply:
If you are using some external resources kindly provide screenshot or video explanation